.github/workflows/pluto_input_cama.yml

```diff
@@ -41,12 +41,33 @@ jobs:
       - name: Run Container Setup
         working-directory: ./
         run: ./bash/docker_container_setup.sh
-        
-      - name: install ssh key
+
+      - name: Setup ssh
+        run: ./pluto ssh setup
+
+      - name: Download file
+        id: download
+        run: |
+          (./pluto ssh download cama) && true
```

Comment:
The `&& true` is a bit hacky, but it makes bash not exit - within pluto cli we use `set -e` so just running this command causes the workflow run to fail